tuplex/codegen/include/AnnotatedAST.h

```diff
@@ -165,7 +164,7 @@ namespace tuplex {
             /*!
              * throw exception for unsupported types
```

Comment:
so this function basically checks the error in IFailable and throws an exception? If so, this should be in the description.

Reply:
Added in comments.